airbyte-integrations/connectors/destination-s3/build.gradle

```diff
@@ -13,7 +13,11 @@ airbyteJavaConnector.addCdkDependencies()
 
 application {
     mainClass = 'io.airbyte.integrations.destination.s3.S3DestinationRunner'
-    applicationDefaultJvmArgs = ['-XX:+ExitOnOutOfMemoryError', '-XX:MaxRAMPercentage=75.0']
+    applicationDefaultJvmArgs = ['-XX:+ExitOnOutOfMemoryError', '-XX:MaxRAMPercentage=75.0', '--add-opens', 'java.base/java.lang=ALL-UNNAMED']
+}
+
+run {
+    standardInput = System.in
```

Comment:
I don't understand this addition.

Reply:
Also for testing, will remove. (It's necessary to make the running gradle `run` task accept input from stdin.)